src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h

```diff
@@ -347,14 +358,23 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
   }
 
   [[nodiscard]] const std::string& GetFunctionName(SFID sfid) const {
-    // TODO(b/247072330) make it configurable
-    return report_.GetSfidToSymbols().at(sfid).baseline_function_symbol->function_name;
+    const orbit_mizar_base::BaselineAndComparisonFunctionSymbols& symbols =
+        report_.GetSfidToSymbols().at(sfid);
+    switch (function_name_to_show_) {
+      case FunctionNameToShow::kBaseline:
+        return symbols.baseline_function_symbol->function_name;
+      case FunctionNameToShow::kComparison:
+        return symbols.comparison_function_symbol->function_name;
+      default:
+        ORBIT_UNREACHABLE();
```

Comment:
I do not like switch default cases. If now someone add a new case to `FunctionNameToShow`, you dont get a compile time error here, but a runtime crash. Please remove the default case and then you probably need to move the `ORBIT_UNREACHABLE()` to after the switch. (Maybe you can completely omit it, but I think some of our compilers need it.)

Reply:
Great point, thank you!